### ISSUE TITLE
Связь между датами агрегации и закрытия смены

### ISSUE THREAD

**Issue by FlyOverTheSky** [axon-expert/backend-test-task#2]
Если смена уже закрыта, те СтатусЗакрытия: True, то можно ли агрегировать продукцию в этой смене? Просто если можно тогда же получится, что агрегировали продукцию после закрытия смены, что нелогично. А если нельзя, то возвращение ошибки 400 с описанием: batch is closed, подойдет?

**Comment by hirotasoshu** [comment]
@FlyOverTheSky да, все верно, в данном случае нужно вернуть 400, ваше описание ошибки корректное)
